api/v1beta2/foundationdbbackup_types.go

```diff
@@ -97,6 +97,8 @@ type FoundationDBBackupSpec struct {
 	// This is the configuration of the target blobstore for this backup.
 	BlobStoreConfiguration *BlobStoreConfiguration `json:"blobStoreConfiguration,omitempty"`
 
+	EncryptionKeyPath string `json:"encryptionKeyPath,omitempty"`
```

Comment:
Could you please add a description for the new variable? We should also add a length validation for this new field (you can find some examples in `FoundationDBCluster`.